nixos/tests/installer.nix

```diff
@@ -682,20 +686,32 @@ in {
     createPartitions = ''
       machine.succeed(
           "flock /dev/vda parted --script /dev/vda -- mklabel msdos"
-          + " mkpart primary linux-swap 1M 1024M"
-          + " mkpart primary 1024M -1s",
+          + " mkpart primary 1M 100MB"  # bpool
```

Comment:
Looks good for this PR, but maybe we should start considering not testing to boot from zfs at all? I think the grub integration for zfs is not very well maintained and we mention this in no installation guide for zfs on NixOS. Given that most people now use EFI on their regular installations, we might just test this use-case instead?

Reply:
Ok. Just read the discussion above. I think we sort of agree, but let's just use your solution for now.